test/reducers/test-removeDeleted.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
Very nice. Love to see additional tests 👍

Reply:
Well, I had an error in that and I had no idea why, so had to figure out what it was doing.